packages/react/src/components/AccountSettings/ConfigureTOTP/types.ts

```diff
@@ -0,0 +1,10 @@
+export interface ConfigureTOTPProps {
+  /** TOTP issuer. Default to "AWSCognito" */
+  totpIssuer?: string;
+  /** TOTP username. Default to  */
```

Comment:
What should it default to?

Reply:
Oh man, I screwed this up didn't I 😓 

it defaults to `user.username`, which is taken from the `user` object.
```suggestion
  /** TOTP username. Default to `user.username`  */
```